environment.yml

```diff
@@ -5,6 +5,7 @@ channels:
 dependencies:
   - python=3.9.7
   - cartopy=0.20.1
+  - cmip6_preprocessing
```

Comment:
Please pin `cmip6_preprocessing` to the known and tested version from the image we're using.

Reply:
Looks like its version in the hub is 0.0.0, I will add that in now!